packages/mobile/src/screens/HomeScreen.tsx

```diff
@@ -56,11 +56,26 @@ const HomeScreen = ({navigation}:any) => {
                 />
               </View>
             </View>
-          </ScrollView>
-        </SafeAreaView>
-      </>
-    );
-  }
+            <View style={styles.sectionContainer}>
```

Comment:
co to ? usuwaliśmy to juz, usun dodatkowe tresci tak zeby zostały tylko te aktualne rzeczy

Reply:
co to?